app/service/app_svc.py

```diff
@@ -170,6 +174,16 @@ async def watch_ability_files(self):
                     await self.get_service('data_svc').load_ability_file(filename=f, access=p.access)
             await asyncio.sleep(int(self.get_config('ability_refresh')))
 
+    def register_subapp(self, path: str,  app: web.Application):
+        """Registers a web application under the root application.
+
+        Requests under `path` will be routed to this app.
+        """
+        self.application.add_subapp(path, app)
+
+    def get_loaded_plugins(self):
+        return tuple(self._loaded_plugins)
```

Comment:
I considered changing this to a generator as well. Might still do that, and potentially make it a read-only property. Just being defensive to avoid unwanted mutation side effects.